components/producers/semgrep/main_test.go

```diff
@@ -25,7 +25,11 @@ const exampleOutput = `
 			"extra": {
 				"message": "Use of this type presents a security risk: the encapsulated content should come from a trusted source, \nas it will be included verbatim in the template output.\nhttps://blogtitle.github.io/go-safe-html/\n", 
 				"metavars": {},
-				"metadata": {}, 
+				"metadata": {
+					"cwe": [
+						"CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')"
```

Comment:
can we add a second entry over here to ensure that the code works as expected in the case of a list with multiple entries